### ISSUE TITLE
Passing console.log.bind(...) to queueMicrotask, setTimeout and others might result in missing log messages

### ISSUE THREAD

**Issue by berbant** [Tampermonkey/tampermonkey#1408]
In version 4.13 and earlier the queueMicrotask command worked well, but in 4.14 this command no longer works in my scripts.
No warnings or errors, just blank string.

## Specifications
- Chromium: (98.0.4758.87)

## Script

```js
// ==UserScript==
// @name         TESTqueueMicrotask
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  try to take over the world!
// @author       You
// @match        *://github.com/*
// @grant        none
// ==/UserScript==

(function() {
    'use strict';
    console.log("TEST") // works
    queueMicrotask (console.log.bind (console, "TEST")); // doesn't work in 4.14
})();
```

**Comment by derjanb** [comment]
The script is work fine here using TM 4.14 and Version 98.0.4758.54 (Official Build) beta (64-bit):

![image](https://user-images.githubusercontent.com/767504/152317409-108b59b9-4d6b-4cb7-a037-f8cc5ea45daa.png)

**Comment by derjanb** [comment]
Ah wait, I see the problem, but is has nothing to do with the queueMicrotask, which is working fine.

```JS
// ==UserScript==
// @name         TESTqueueMicrotask
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  try to take over the world!
// @author       You
// @match        *://github.com/*
// @grant        none
// ==/UserScript==

(function() {
    'use strict';
    console.log("TEST") // works
    queueMicrotask (() => console.log("TEST2")); // doesn't work in 4.14
})();
```

**Comment by berbant** [comment]
[derjanb](https://github.com/derjanb), you are right, queueMicrotask is working in itself but there is a trick to hide some garbage from console window: https://stackoverflow.com/a/67026569 and this combination stopped working in this version TM.

**Comment by derjanb** [comment]
Should be fixed at 4.15.6153 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_4_15_6153.crx))
